tests/unit/panther_analysis_tool/test_enriched_event_generator.py

```diff
@@ -25,7 +23,6 @@
 )
 
 
-@nottest
```

Comment:
Is this not needed now?

Reply:
Correct, `nose2` doesn't have that decorator: https://docs.nose2.io/en/latest/decorators.html

Reply:
That decorator was used to tell nose that function isn't a test a shouldn't be treated as such. How do we replicate that functionality in nose2?

Reply:
Looks like we can use `@unittest.skip` -- I'll push that change.

Reply:
I moved the code to its own module in [`8cf5785` (#493)](https://github.com/panther-labs/panther_analysis_tool/pull/493/commits/8cf5785538d8d2a5f2d9347c25b9c8b731d744e6). 👍🏻